### PR TITLE
v4.0.0

### PR DESCRIPTION
**Pull request by davidfloegel**
## Breaking Changes

The `color` prop on the `<Divider />`, `<Typography />` and `<Icon />` component used to accept a hexcode for coloring. This has now been changed so that only colors from the theme can be used.

**Updated usages**

```diff
- <Divider color="#111111" />
+ <Divider color="backToBlack" />

- <H1 color="red">I am a red H1</H1>
+ <H1 color="redRedWine">I am a red H1</H1>

- <Icon color="#039842" name="chevronUp" />
- <Icon color="primary" name="chevronUp" />
```

### The Grid `<Column />` now has a `rowSpan` prop

The Grid has been extended to have a `rowSpan` attribute which allows the developer to span one column across multiple rows.

**Usage**
```
<Grid>
  <Col xs={12} sm={6} rowSpan={2}>I will take 2 rows in height.</Col>
  <Col xs={12} sm={6}>I will take 1 row in height.</Col>
</Grid>
```

## Bug Fixes

- Fix styling issue on `<FormGroup />` component in error state
- Add correct transparent color on `<LinkButton />` and `<OutlineButton />` on `&:focus` state

### CODE REVIEW

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/187
Branch:
https://sofarsounds.github.io/maestro/branches/develop
Storybook:
https://sofarsounds.github.io/maestro/7682d2e104ecae7e6d5cf280759762a1e2dbacc6/
Build:
https://sofarsounds.github.io/maestro/7682d2e104ecae7e6d5cf280759762a1e2dbacc6/dist.zip

File: src/atoms/Divider/index.tsx [hunk 1]
```diff
@@ -2,12 +2,13 @@ import styled, { css } from 'styled-components';
...
   space?: number;
+  color?: string;
 }
 
 export default styled.hr<DividerProps>`
-  ${({ theme, space }) => css`
+  ${({ theme, space, color }) => css`
     border: none;
-    border-top: 1px solid ${theme.colors.silverSprings};
+    border-top: 1px solid ${color || theme.colors.silverSprings};
```
Comment (liondedan) [inline src/atoms/Divider/index.tsx]: This color prop should come from theme ^

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/187
Branch:
https://sofarsounds.github.io/maestro/branches/develop
Storybook:
https://sofarsounds.github.io/maestro/0036cbe4c754ec19b47d1dee92df299b70295322/
Build:
https://sofarsounds.github.io/maestro/0036cbe4c754ec19b47d1dee92df299b70295322/dist.zip

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/187
Branch:
https://sofarsounds.github.io/maestro/branches/develop
Storybook:
https://sofarsounds.github.io/maestro/fcab5f9febe1ea419c334128c9f7f784ad8edb23/
Build:
https://sofarsounds.github.io/maestro/fcab5f9febe1ea419c334128c9f7f784ad8edb23/dist.zip

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/187
Branch:
https://sofarsounds.github.io/maestro/branches/develop
Storybook:
https://sofarsounds.github.io/maestro/58f45f7736c5a59e6bc7adb577f1c279902b99ed/
Build:
https://sofarsounds.github.io/maestro/58f45f7736c5a59e6bc7adb577f1c279902b99ed/dist.zip

**Pull request merged**